### ISSUE TITLE
This repository contains malware.

### ISSUE THREAD

**Issue by hexaheximal** [PythonWell/Discord-Mass-DM-Bot-by-PythonWell#22]
# This repository contains malware.

I was recently told that this repository contains malware in [install.bat](https://github.com/PythonWell/Discord-Mass-DM-Bot-by-PythonWell/blob/main/install.bat), and I decided to look at it myself.

I already know that the software in this repository is **against the Discord ToS**, and I have not tried to use it because of that.

The malware in [install.bat](https://github.com/PythonWell/Discord-Mass-DM-Bot-by-PythonWell/blob/main/install.bat) is in multiple stages of Encrypted & Compressed C# code, but here's the native payload it runs so anyone who wants to reverse-engineer this malware can skip multiple stages and just reverse-engineer the payload:
https://github.com/JuicySniper/juicy/raw/main/wininfo64.exe

**I reccommend reporting https://github.com/JuicySniper/juicy, because this malware will become useless if that repository gets deleted** (because it downloads the payload from it)

In the [instructions list](https://github.com/PythonWell/Discord-Mass-DM-Bot-by-PythonWell#instructions), it says the following:

```
Start install.bat to install dependencies or this won't work.
```

**That is not true.**
It's really asking you to run it because it contains malware that the author of this software (PythonWell) is using to steal Discord accounts. In fact, I have known that before reverse-engineering it.

I also noticed that [most of the non-malware code in this repository was taken from other people](https://github.com/PythonWell/Discord-Mass-DM-Bot-by-PythonWell/issues/19) and that the [requirements list](https://github.com/PythonWell/Discord-Mass-DM-Bot-by-PythonWell#requirements) requires a `Windows Machine` ***because it's what the malware code was designed for***

The non-malware code should work on Linux and MacOS (maybe even BSD if you have too much time), but the requirements list says that it runs on Windows specifically, ***because the malware code only runs on Windows***.

**TL;DR DO NOT RUN INSTALL.BAT!**
